lib/security/WSSecurity.js

```diff
@@ -80,11 +80,17 @@ WSSecurity.prototype.toXML = function() {
     "xmlns:wsse=\"http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd\" xmlns:wsu=\"http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd\">" +
     timeStampXml +
     "<wsse:UsernameToken xmlns:wsu=\"http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd\" wsu:Id=\"SecurityToken-" + created + "\">" +
-    "<wsse:Username>" + this._username + "</wsse:Username>" +
+    "<wsse:Username>" + wrapInCdata(this._username) + "</wsse:Username>" +
     password +
     (this._hasTokenCreated ? "<wsu:Created>" + created + "</wsu:Created>" : "") +
     "</wsse:UsernameToken>" +
     "</wsse:Security>";
 };
 
+function wrapInCdata(value) {
```

Comment:
could you please move the declaration of your function onto the top of this file, so that it's declared before we use it? thanks 👍